docs/08-developer-guide/01-build/01-maven.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_label: Maven
+---
+
+
+# Building Ozone With Maven
+
+- Cover basic Maven commands to build and run tests.
+- Document all the Ozone specific Maven flags we use to speed up or skip parts of the build, and when they are useful.
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide explains how to build Apache Ozone from source using Maven and prepare it for deployment.
+
+## Prerequisites
+
+**TODO** : [HDDS-11625](https://issues.apache.org/jira/browse/HDDS-11625) Finalize the version numbers of prerequisite packages
```

Comment:
I think the existing Jira is enough to track this. We can commit the docs page free of TODOs.